lib/LaTeXML/Package.pm

```diff
@@ -1369,12 +1369,12 @@ sub dualize_arglist {
       push(@cargs, $arg); }
     elsif ($used{$i}) {                            # used in presentation?
       my $id = getXMArgID();
-      push(@pargs, Invocation(T_CS('\@XMArg'), $id, $arg));    # put XMArg in presentation
-      push(@cargs, Invocation(T_CS('\@XMRef'), $id)); }
+      push(@pargs, Invocation(T_CS('\lx@xmarg'), $id, $arg));    # put XMArg in presentation
+      push(@cargs, Invocation(T_CS('\lx@xmref'), $id)); }
     else {                                                     # Hidden arg, put XMArg in content.
       my $id = getXMArgID();
-      push(@cargs, Invocation(T_CS('\@XMArg'), $id, $arg));
-      push(@pargs, Invocation(T_CS('\@XMRef'), $id)); } }
+      push(@cargs, Invocation(T_CS('\lx@xmarg'), $id, $arg));
+      push(@pargs, Invocation(T_CS('\lx@xmref'), $id)); } }
```

Comment:
Hey, at least they're lowercase! I suppose `\lx@arg` and `\lx@ref` were too generic for you to feel safe with?